README.md

```diff
@@ -2,5 +2,30 @@ cosmoz-bottom-bar
 ==================
 
 [![Build Status](https://travis-ci.org/Neovici/cosmoz-bottom-bar.svg?branch=master)](https://travis-ci.org/Neovici/cosmoz-bottom-bar)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://beta.webcomponents.org/element/PolymerElements/paper-button)
 
-The Polymer element **cosmoz-bottom-bar** is a responsive bottom-bar that can house buttons/actions and a menu for the buttons that won't fit the available width.
\ No newline at end of file
+## &lt;cosmoz-bottom-bar&gt;
+
+**cosmoz-bottom-bar** is a responsive bottom-bar that can house buttons/actions and a menu for the buttons that won't fit the available width.
+
+Example:
+
+<!---
+```
+<custom-element-demo>
```

Comment:
I still don't get this to work in preview.

I need to do something like this:

```
<!---
\```
<custom-element-demo>
  <template>
    <script src="../webcomponentsjs/webcomponents-lite.js"></script>
    <link rel="import" href="cosmoz-bottom-bar.html">
    <link rel="import" href="../paper-button/paper-button.html">
    <next-code-block></next-code-block>
  </template>
</custom-element-demo>
\```
-->
```html
<cosmoz-bottom-bar active>
  <paper-button>Action 1</paper-button>
  <paper-button>Action 2</paper-button>
  <paper-button>Action 3</paper-button>
  <paper-button>Action 4</paper-button>
  <paper-button>Action 5</paper-button>
</cosmoz-bottom-bar>
\```
```

(Except for the 3 `\`)

Not sure why the bottom-bar isn't loading in your code though, might be a bug with the preview.
But maybe switch to this example otherwise.